includes/class-rest.php

```
@@ -244,7 +244,9 @@ private function sanitize_settings_recursively( $settings ) {
 				} elseif ( is_numeric( $value ) ) {
 					$sanitized[ $safe_key ] = (int) $value;
 				} elseif ( stripos( $safe_key, 'api_key' ) !== false || stripos( $safe_key, 'password' ) !== false ) {
```

PHPStan (2.2.5)

[warning] Parameter `#1` $haystack of function stripos expects string, string|null given.


(argument.type)